data.sql

```diff
@@ -1,5 +1,19 @@
-insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
-values ('Agumon', '2020-02-03', 0, '1', 10.23),
-('Gabumon', '2018-11-15', 2, '1', 8),
-('Pikachu', '2021-01-7', 1, '0', 15.04),
-('Devimon', '2017-05-12', 5, '1', 11);
+insert into animals 
+  (name, date_of_birth, escape_attempts, neutered, weight_kg)
+values 
+  ('Agumon', '2020-02-03', 0, '1', 10.23),
+  ('Gabumon', '2018-11-15', 2, '1', 8),
+  ('Pikachu', '2021-01-7', 1, '0', 15.04),
+  ('Devimon', '2017-05-12', 5, '1', 11);
+
+insert into animals 
+	(name, date_of_birth, escape_attempts, neutered, weight_kg)
+values 
+	('Charmander', '2020-02-08', 0, '0', 11),
+	('Plantmon', '2021-11-15', 2, '1', 5.7),
+	('Squirtle', '1993-04-02', 3, '0', 12.13),
+	('Angemon', '2005-06-12', 1, '1', 45),
```

Comment:
- [ ] Kindly make sure that you inserted the correct value for the Animal data some of the values are negative value but so insert the value with the positive values. Kindly Fix the issues.